nautobot/extras/tests/test_views.py

```diff
@@ -2766,7 +2769,48 @@ def test_list_objects_with_constrained_permission(self):
         self.assertNotIn(instance2.destination.name, content, msg=content)
 
 
+class StaticGroupTestCase(ViewTestCases.PrimaryObjectViewTestCase):
+    model = StaticGroup
+
+    @classmethod
+    def setUpTestData(cls):
+        content_type = ContentType.objects.get_for_model(Device)
+
+        cls.form_data = {
+            "name": "The Best Devices",
+            "description": "No really!",
+            "content_type": content_type.pk,
+            "tenant": Tenant.objects.first().pk,
+        }
+
+        cls.bulk_edit_data = {
+            "description": "Is anyone there?",
+            "tenant": Tenant.objects.last().pk,
+        }
+
+    def test_edit_object_with_permission(self):
+        instance = self._get_queryset().first()
+        self.form_data["content_type"] = instance.content_type.pk  # Content-type is not editable after creation
+        super().test_edit_object_with_permission()
+
+    def test_edit_object_with_constrained_permission(self):
+        instance = self._get_queryset().first()
+        self.form_data["content_type"] = instance.content_type.pk  # Content-type is not editable after creation
+        super().test_edit_object_with_constrained_permission()
+
+
+class StaticGroupAssociationTestCase(
+    ViewTestCases.BulkDeleteObjectsViewTestCase,
+    ViewTestCases.DeleteObjectViewTestCase,
+    ViewTestCases.GetObjectViewTestCase,
+    ViewTestCases.GetObjectChangelogViewTestCase,
+    ViewTestCases.ListObjectsViewTestCase,
```

Comment:
Is there any other way to access these features besides manually entering `extras/static-group-associations/` in the browser?

Reply:
I *think* the changelog view on the home page might link to the detail view for a StaticGroupAssociation, but other than that, I don't believe so. Kinda similar to RelationshipAssociation in that regard.